src/Compilers/Test/Core/Traits/WorkItemTraitDiscoverer.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Linq;
+using Xunit.Abstractions;
+using Xunit.Sdk;
+
+namespace Microsoft.CodeAnalysis.Test.Utilities;
+
+public sealed class WorkItemTraitDiscoverer : ITraitDiscoverer
```

Comment:
> Allows searching for tests marked with WorkItemAttribute, e.g., in VS Test Explorer via Trait:"...issue url..."

Could you give an example/screenshot of what the experience looks like once the discoverer is in place?

Reply:
Added screenshot to PR description.